helm-chart/amalthea/templates/deployment.yaml

```diff
@@ -66,6 +66,12 @@ spec:
                 {{- include "amalthea.selectorLabels" . | nindent 16 }}
             - name: EXTRA_CHILD_RESOURCES
               value: {{ .Values.extraChildResources | toJson | quote }}
+            - name: SESSION_IDLE_CHECK_INTERVAL_SECONDS
```

Comment:
This is a bit picky, but SESSION is somewhat of a Renku terminology, so I'd either omit `SESSION_` it completely or use `JUPYTER_SERVER_` instead.